FileLiberator/DownloadDecryptBook.cs

```diff
@@ -11,10 +11,11 @@
 using AaxDecrypter;
 using AudibleApi;
 
-namespace FileLiberator.AaxcDownloadDecrypt
```

Comment:
Thanks. I wasn't going to mention this one but it did strike me as a different feel from the others